dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNIIDMExtensions.hpp

```diff
@@ -78,6 +78,10 @@ class IIDMExtensions {
    */
   template<class T>
   static ExtensionDefinition<T> getExtension(const std::string& libPath) {
+    if (libPath.empty()) {
+      // This corresponds to default case if environement variable for external IIDM extensions paths is not set
```

Comment:
typo environment

Reply:
maybe we should add a warning or a debug log in this case?

Reply:
same reason as below, in catch. This would pollute the log as this function is called for every element supporting an external IIDM extension (Lines, generators, HVDC lines)